src/Pages/Login/Login.scss

```
@@ -0,0 +1,190 @@
+      border: 1px solid #000;
+      border-top: 3px solid #000;
+
+      h1 {
+        margin-top: 45px;
+        padding: 0 0 5px;
+        font-size: 26px;
+      }
+
+      .loginForm {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 420px;
+        margin-top: 100px;
+
+        label {
+          width: 20px;
+          height: 25px;
+          border: 1px solid #000;
+          padding: 8px 12px;
+          border-right: none;
+
+          #idIcon {
+            font-size: 23px;
+            text-align: center;
+          }
+        }
+
+        .idInput {
+          width: 350px;
+          height: 25px;
+          border: 1px solid #000;
+          border-left: none;
+        }
+      }
+
+      .pwForm {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 420px;
+        margin-top: 10px;
+
+        .pwIcon {
+          width: 20px;
+          height: 25px;
+          border: 1px solid #000;
+          padding: 8px 12px;
+          border-right: none;
+
+          #pwIcon {
+            font-size: 23px;
+            text-align: center;
+          }
+        }
+
+        .pwInput {
+          width: 350px;
+          height: 25px;
+          border: 1px solid #000;
+          border-right: none;
+          border-left: none;
+        }
+
+        .show {
+          width: 20px;
+          height: 25px;
+          border: 1px solid #000;
+          padding: 8px 12px;
+          border-left: none;
+          font-size: 17px;
+          cursor: pointer;
+        }
+      }
+
+      .Remember {
+        display: flex;
+        justify-content: space-between;
+        width: 420px;
+        margin-top: 30px;
+
+        .rememberId {
+          display: flex;
+          align-items: center;
+        }
+
+        .forgetPw {
+          span {
+            color: #4317fc;
+          }
+        }
+      }
+
+      .loginButton {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin-top: 35px;
+
+        button {
+          color: #fff;
+          font-size: 18px;
+          width: 420px;
+          height: 50px;
+          background-color: #3bb271;
+          cursor: pointer;
+        }
+
+        button:hover {
+          background-color: rgb(0, 138, 0);
+        }
+      }
+    }
+
+    .signupCont {
+      display: flex;
+      width: 490px;
+      height: 500px;
+      border: 1px solid #000;
+
+      .signupForm {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        margin-top: 45px;
+        margin-left: 37px;
+
+        h1 {
+          font-size: 26px;
+        }
+
+        .benifit {
+          display: flex;
+          flex-direction: column;
+          margin-top: 30px;
+          text-align: center;
+        }
+      }
+      .Logo {
+        margin-top: 60px;
+        width: 100px;
+      }
+      .SignupBtn {
```

```suggestion
      }
      
      .Logo {
        margin-top: 60px;
        width: 100px;
      }
      
      .SignupBtn {
```
새로운 셀렉터가 시작될 때는 가독성 차원에서 한 줄씩 띄워주세요.